script.js

```diff
@@ -11,4 +13,328 @@ hamburger.addEventListener('click', toggleMenu);
+  },
+
+];
+
+for (let i = 0; i < projects.length; i++) {
+  const project = document.createElement('article');
+  project.classList.add('project');
+  x.appendChild(project);
+
+  const image = document.createElement('article');
+  image.classList.add('project-img');
+  project.appendChild(image);
+
+  const title = document.createElement('h3');
+  title.classList.add('project-title');
+  title.innerHTML = projects[i].title;
+  project.appendChild(title);
+
+  const languages = document.createElement('ul');
+  for (let j = 0; j < projects[i].languages.length; j++) {
+    const language = document.createElement('li');
+    language.innerHTML = projects[i].languages[j];
+    languages.appendChild(language);
+  }
+  project.appendChild(languages);
+
+  const button = document.createElement('button');
+  button.setAttribute('id', 'project-button');
+  button.classList.add('button');
+  button.innerHTML = projects[i].buttonName;
+  project.append(button);
+}
+
+function createPopup(index) {
+  const project = projects[index];
+  const x = document.querySelector('.works');
+
+  const overlay = document.createElement('div');
+  overlay.classList.add('overlay');
+  x.appendChild(overlay);
+
+  const popupWindow = document.createElement('div');
+  popupWindow.setAttribute('id', 'popup-window');
+  popupWindow.classList.add('popup-section');
+
+  const popupCloseButton = document.createElement('button');
+  popupCloseButton.setAttribute('id', 'close-button');
+  popupCloseButton.classList.add('close-button');
+  popupCloseButton.innerHTML = '&times';
+  popupWindow.appendChild(popupCloseButton);
+
+  const popupImageDesk = document.createElement('img');
+  popupImageDesk.setAttribute('src', project.popupImageDesktop);
+  popupImageDesk.classList.add('desktop', 'popup-image');
+  popupWindow.appendChild(popupImageDesk);
+
+  const popupImageMob = document.createElement('img');
+  popupImageMob.setAttribute('src', project.popupImageMobile);
+  popupImageMob.classList.add('mobile', 'popup-image');
+  popupWindow.appendChild(popupImageMob);
+
+  const popupTitleButtonsContainer = document.createElement('div');
+  popupTitleButtonsContainer.classList.add('title-buttons');
+  popupWindow.appendChild(popupTitleButtonsContainer);
+
+  const popupTitle = document.createElement('h3');
+  popupTitle.classList.add('popup-title');
+  popupTitle.innerHTML = project.popupTitle;
+  popupTitleButtonsContainer.appendChild(popupTitle);
+
+  const popupButtonsContainerDesktop = document.createElement('div');
+  popupButtonsContainerDesktop.classList.add('desktop', 'popup-buttons');
+  popupTitleButtonsContainer.appendChild(popupButtonsContainerDesktop);
```

Comment:
👏 Thank you for your efforts in adjusting the position of the buttons to match the design in the desktop version as the previous reviewer requested! However, I noticed that there are still some refinements needed as it arise because of the changes you made after the previous review's request. Specifically, there appears to be extra margin on the left and right sides of your title and button elements. Upon closer examination of your code, I identified that this is related to a CSS rule on line number 546 that affects all elements:

```css
* {
    margin: 0 auto;
}
```

- [ ] This global rule is causing the additional margin within the `title-buttons` wrapper div. While I understand that you have applied this rule for a consistent layout, making changes here might impact other parts of the design. To address this, I recommend overriding the general rule by specifically adding `margin-left: 0` and `margin-right: 0` in the `.title-buttons` class. This targeted adjustment should help align the title and buttons properly without affecting other elements.

| Your | Design |
| --- | --- |
| ![Frame 45](https://github.com/danielf-canon/Portfolio/assets/55840999/b0bddd7a-487f-42a9-be64-dac5877b13b9) | ![Frame 44 (1)](https://github.com/danielf-canon/Portfolio/assets/55840999/3ec7f1ac-7062-42e3-8b4b-e69de894e3a3) |


By making this adjustment, you will ensure that the spacing within the `title-buttons` wrapper matches the design requirements. Keep up the good work, and happy coding! 🚀💻🌟